<launch>

  <!-- Arguments -->
  <arg name="name" default="json_data_handler"/>
  <arg name="input_topic" default="webrtc_data"/>

  <node name="$(arg name)" pkg="opentera_webrtc_ros" type="json_data_handler" output="screen">
    <remap from="webrtc_data" to="$(arg input_topic)"/>
  </node>
  
</launch>